### DISCUSSION TITLE
Restating Importance of PageRank Alternatives

### DISCUSSION THREAD

**Discussion by BradKML** [networkx/networkx#5062] [Algorithms]
The Current PageRank algorithms are not robust against Link Farms (dense sub-networks), Link Bombing (heavy citation abuse) and Mutual Admiration Societies (Bi-directional Links). Also, PageRank is currently unsuitable for citation graphs (and other DAGs)
http://cs.wellesley.edu/~pmetaxas/Metaxas-Evolution_Search_Engine_Rankings.pdf

- ArticleRank (Citation Graph and DAG application with some factors changed)
  - https://eprints.whiterose.ac.uk/10323/1/Willett_10323.pdf
  - https://github.com/neo4j-contrib/neo4j-graph-algorithms/issues/689
  - Neo4j has it https://neo4j.com/docs/graph-data-science/current/algorithms/article-rank/
- BackRank (adds a factor allowing for walk-backs in websites without external links)
  - http://delab.csd.auth.gr/~dimitris/courses/ir_spring06/page_rank_computing/p1122-bouklit.pdf
- Truncated PageRank, SpamRank and Parent Pentalty (spam detection)
  - http://sewm.pku.edu.cn/TianwangLiterature/2010/2010-JICS-A%20Survey%20of%20Major%20Techniques%20for%20Combating%20Link%20Spamming.pdf
  - https://www.sogou.com/labs/paper/R-SpamRank_A_Spam_Detection_Algorithm_Based_on_Link_Analysis.pdf
- Others
  - PageRank-Index (impact of researcher) https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4545754/

**Comment by dschult** [comment]
There are so many definitions of rank that basically just use PageRank.  
Are these "measures" of rank different from the algorithms we provide, or are they applications of what we provide?

**Reply by BradKML** [reply]
@dschult I rewrote the subject in regards to centrality that factors out community bias, and makes it more "fair" for citation and importance.

**Comment by BradKML** [comment]
More Citation Graph Network Centralities
- Bi-directional h-index https://ideas.repec.org/a/eee/infome/v12y2018i1p299-314.html
- VoteRank https://arxiv.org/ftp/arxiv/papers/1811/1811.01638.pdf
- Complex Degree Centrality http://eprints.rclis.org/7606/2/Kretschmer18aps.pdf
- Stress Centrality https://sci-hub.se/https://ieeexplore.ieee.org/document/1690659
